package.json

```diff
@@ -1,49 +1,32 @@
 {
   "name": "react-example-project",
-  "version": "0.1.0",
   "private": true,
   "license": "MIT",
-  "dependencies": {
-    "@nordhealth/components": "*",
-    "@nordhealth/css": "*",
-    "@nordhealth/react": "*",
-    "@testing-library/jest-dom": "^5.16.2",
-    "@testing-library/react": "^14.0.0",
-    "@testing-library/user-event": "^13.5.0",
-    "@types/jest": "^27.4.1",
-    "@types/node": "^16.11.26",
-    "@types/react": "^17.0.39",
-    "react": "^18.1.0",
-    "react-dom": "^18.1.0",
-    "react-scripts": "^5.0.1",
-    "typescript": "^4.6.2",
-    "web-vitals": "^2.1.4"
-  },
+  "version": "0.2.0",
+  "type": "module",
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "dev": "vite",
+    "build": "tsc -b && vite build",
+    "lint": "eslint .",
+    "preview": "vite preview"
   },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
+  "dependencies": {
+    "react": "^19.1.0",
+    "react-dom": "^19.1.0",
+    "@nordhealth/components": "^4.1.0",
+    "@nordhealth/css": "^4.0.0"
```

Comment:
Instead of referencing specific versions here, I’d rather suggest doing what we did here previously (e.g. `*`) to make sure the consumer always gets the latest versions when creating a project with this boilerplate.

Reply:
I thought about that, but I think that's tricky if they use this template but then don't adjust it to a semver range, then they will automatically get breaking changes. Plus, it makes it clear on what version we've tested this. The README also specifies we recommend doing those steps yourselves to get the latest versions instead of using the template directly. But I'm ok with using `*` again if you want. They both have their pros and cons.

Reply:
Let’s go with your recommendation, thanks Elwin!